grid-cli/internal/cmd/get.go

```diff
@@ -8,6 +8,24 @@ import (
 	"github.com/threefoldtech/tfgrid-sdk-go/grid-client/workloads"
 )
 
+func checkIfExistAndAppend(t deployer.TFPluginClient, node uint32, contractID uint64) {
+
+	if len(t.State.CurrentNodeDeployments[node]) != 0 {
+
+		for _, n := range t.State.CurrentNodeDeployments[node] {
+
+			if n != contractID {
```

Comment:
this will append always even more than once for each contract ID :)
better check if n == contractID and if matched just return